README.md

```diff
@@ -18,12 +18,37 @@ We welcome contributions and feedback. Please follow our [contributing guideline
 
 ---
 
-## To run locally
+## one-time setup
```

Comment:
upper case
this should be any time you pull changes, not one-time setup
the dev-cert can be one-time setup

Reply:
I changed this to say run npm install after pulling changes. got rid of the step I added that says to clone the repo, because it should be obvious

Reply:
yeah, it might be clearer to have npm install in each section